test/TransitionGroup-test.js

```diff
@@ -13,7 +13,7 @@ describe('TransitionGroup', () => {
   beforeEach(() => {
     React = require('react');
     ReactDOM = require('react-dom');
-    Transition = require('../src/Transition').default;
+    Transition = require('../src/Transition');
```

Comment:
This is the only breaking change (for cjs `require` users) I can see here, so they won't have to type `.default` for `Transition` anymore. That how it should have worked from the beginning, though.